lib/chat/ui/widgets/styles/bubble_message_styles.dart

```diff
@@ -2,55 +2,121 @@ import 'package:flutter/material.dart';
-  static const colorBlack = const Color(0xFF1B1B34);
-  static const colorWhiteAccented = const Color(0xFFF5F5FA);
-  static const colorGreen = const Color(0xFF00CD9F);
-  static const colorWhite = const Color(0xFFFFFFFF);
+  static const receivedTextColor = const Color(0xFF1A1B46);
+  static const receivedBackgroundColor = const Color(0xFFE9EAF2);
+  static const sentTextColor = const Color(0xFFFFFFFF);
+  static const sentBackgroundColor = const Color(0xFF24D900);
+  static const defaultRadius = Radius.circular(20.0);
+
+  static const messageAdditionalMargin = 48.0;
+  static const spaceBetweenMessages = 24.0;
 }
 
 class MessageBubbleStyles {
   static MessageBubbleStyle buildStyleSent() => _defaultMolecule;
 
   static MessageBubbleStyle buildStyleReceived() => _defaultMolecule.copyWith(
-        textStyle:
-            const TextStyle(color: _Constants.colorBlack, fontSize: 15.0),
-        textContainerBackgroundColor: _Constants.colorWhiteAccented,
+        textStyle: const TextStyle(
+          color: _Constants.receivedTextColor,
+          fontSize: 15.0,
+        ),
+        outerContainerMargin: EdgeInsets.only(
+          right: _Constants.messageAdditionalMargin,
+          top: _Constants.spaceBetweenMessages,
+        ),
+        textContainerBackgroundColor: _Constants.receivedBackgroundColor,
         rowCrossAxisAlignment: CrossAxisAlignment.start,
         rowMainAxisAlignment: MainAxisAlignment.start,
-        textContainerMargin: const EdgeInsets.only(left: 12.0, right: 40.0),
+        textContainerBorderRadius:
+            const BorderRadius.all(_Constants.defaultRadius),
+      );
+
+  static MessageBubbleStyle buildStyleReceivedStackTop() =>
+      buildStyleReceived().copyWith(
+        textContainerBorderRadius: BorderRadius.only(
+          topLeft: _Constants.defaultRadius,
+          topRight: _Constants.defaultRadius,
+          bottomLeft: Radius.circular(5.0),
+          bottomRight: _Constants.defaultRadius,
+        ),
+        outerContainerMargin: EdgeInsets.only(
+          bottom: 4.0,
+          right: _Constants.messageAdditionalMargin,
+          top: _Constants.spaceBetweenMessages,
+        ),
+      );
+
+  static MessageBubbleStyle buildStyleReceivedStackBottom() =>
+      buildStyleReceived().copyWith(
+        textContainerBorderRadius: BorderRadius.only(
+          topLeft: Radius.circular(5.0),
+          topRight: _Constants.defaultRadius,
+          bottomLeft: _Constants.defaultRadius,
+          bottomRight: _Constants.defaultRadius,
+        ),
+        outerContainerMargin: EdgeInsets.only(
+          right: _Constants.messageAdditionalMargin,
+          top: 4.0,
+        ),
+      );
+
+  static MessageBubbleStyle buildStyleReceivedStackBetween() =>
+      buildStyleReceived().copyWith(
+        textContainerBorderRadius: BorderRadius.only(
+          topLeft: Radius.circular(5.0),
+          topRight: _Constants.defaultRadius,
+          bottomLeft: Radius.circular(5.0),
```

Comment:
Maybe we can add this Radius as a Constant too?

Reply:
Moved to constants.